composer.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "digipolisgent/robo-drupal-console",
+    "description": "Drupal Console CommandStack for Robo Task Runner",
+    "type": "robo-tasks",
+    "require": {
+        "consolidation/robo": "~1.0",
+        "drupal/console": "~1.0",
+        "php": ">=5.5.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "~4.4"
+    },
+    "minimum-stability": "dev",
+    "autoload": {
+        "psr-4": {
+            "DigipolisGent\\Robo\\Task\\DrupalConsole\\": "src"
```

Comment:
Is volgens mij geen standard practice om dat te doen, autoload is voor packages die deze includen, testen includen is nu niet echt nodig IMO.
@zero2one thoughts?